core/src/autogluon/core/_setup_utils.py

```diff
@@ -10,7 +10,7 @@
     os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '..', '..')
 )
 
-PYTHON_REQUIRES = '>=3.6, <3.8'
+PYTHON_REQUIRES = '>=3.6, <3.9'
```

Comment:
Given this appears to work and has passed CI, should we merge this PR? Perhaps the ConfigSpace update fixed all the issues people were having with 3.8.

Reply:
@jwmueller @sxjscience @gradientsky

Reply:
I agree with updating the `PYTHON_REQUIRES` to include 3.8. Not sure the CI (build.yml) should be 3.8 or 3.7 though, may still want to keep it at 3.7 if that's where most users are.